appveyor.yml

```diff
@@ -5,31 +5,38 @@ skip_tags: false
 skip_non_tags: false
 skip_branch_with_pr: false
 
-image: Visual Studio 2017
-
-clone_folder: C:\projects\build
 shallow_clone: false
 clone_depth: 5
 
 environment:
   global:
     APPVEYOR_RDP_PASSWORD:
       secure: KN38AnwEZQ6wMS1ktlkybV0BEvTUgnvajLAPBJWdH8I4lttpWzTkLdU+XgLoG6X//t76IbiaIdEPFKEDzgoHyY2xGIxofmS2BDtQD+I9T3o=
+    APPVEYOR_SSH_KEY: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDDG7+Rf4XVbw3QUZwKR8UehNXXKuxgoLnHi4/VQ3w5CrOmJ0ANCoPsm34jJlBUM06Qkxi7LGjZs3TAKmzUKMKkG8K5y7xlHMVkYxrd9HVchrBCLCMB0SdWNZ3uqkmOtwkGCHSyYkXlzHT5qFMbxoJHsLsrXrp9UVTyOVPBZI/7lnmeLq6xofZJ7PjJA//NB0WgjU1Rlri7+Xo2/ZTUdq6i6TqX8oZHWJNCn3q3tPJ/gVU7/6FgxtArduiK8rq45uROAq1Vhhu7vXMgXZaMtzXKwhVx68w9GGLkm8e5cvey1rIaJFj3Gx7NsORwzVwwtIQ3FP1KI3qonmPvBcV/0icb
     MSVC_VERSION: 14.16.27023
     WIN_SDK_VERSION: 10.0.26624.0
     WIN_MSYS2_ROOT: C:\msys64
     WIN_MSYS2_CACHE: C:\msys64\var\cache\pacman\pkg
     WIN_OTP_PATH: C:\Program Files\erl
-    BUILD_PATH: /c/projects/build
+    BUILD_PATH: /c/projects/aeternity
+    ERL_EPMD_ADDRESS: 127.0.0.1
```

Comment:
Why is this necessary? It would be nice to be able to pass this to epmd always, not only in test.

Reply:
On Windows the default behaviour of epmd is somewhat undeterministic and specific to the setup. That's why I fixed it here to locahost.